index.js

```diff
@@ -13,18 +13,43 @@ function run() {
       core.setFailed('parsing error!');
       return;
     }
+
+    const linesMissingCoverage = [];
+
     let totalFinds = 0;
     let totalHits = 0;
     data.forEach((element) => {
       if (shouldCalculateCoverageForFile(element['file'], excludedFiles)) {
-        totalHits += element['lines']['hit'];
         totalFinds += element['lines']['found'];
+        totalHits += element['lines']['hit'];
+
+        for (const lineDetails of element['lines']['details']) {
+          const hits = lineDetails['hit'];
+
+          if (hits === 0) {
+            const fileName = element['file'];
+            const lineNumber = lineDetails['line'];
+            linesMissingCoverage[fileName] =
+              linesMissingCoverage[fileName] || [];
+            linesMissingCoverage[fileName].push(lineNumber);
```

Comment:
nit

```suggestion
            linesMissingCoverage[fileName] =
              (linesMissingCoverage[fileName] || []).push(lineNumber);
```

Reply:
I think you might mean `(linesMissingCoverage[fileName] || [])..push(lineNumber)` with two periods (i.e., cascading). 🤔
Unfortunately, that syntax is not supported it seems.

I think that doing what you proposed, with one period, would set `linesMissingCoverage[fileName]` to a `number` since `.push` returns the index of the new element. Though I might be wrong! :)

Reply:
You could do:
```suggestion
            linesMissingCoverage[fileName]?.push(lineNumber);
```

Reply:
That would just not perform the push operation at all if the list at that `fileName` index doesn't exist yet, right? 🤔 
Which would result in an empty map.

Or am I missing something? 🕵🏻

Reply:
Oh you're totally right 🤦🏻